src/base/bittorrent/session.cpp

```diff
@@ -1538,13 +1538,12 @@ void Session::processShareLimits()
             if (torrent->seedingTimeLimit() != TorrentHandle::NO_SEEDING_TIME_LIMIT) {
                 const qlonglong seedingTimeInMinutes = torrent->seedingTime() / 60;
                 int seedingTimeLimit = torrent->seedingTimeLimit();
-                if (seedingTimeLimit == TorrentHandle::USE_GLOBAL_SEEDING_TIME)
+                if (seedingTimeLimit == TorrentHandle::USE_GLOBAL_SEEDING_TIME) {
                      // If Global Seeding Time Limit is really set...
                     seedingTimeLimit = globalMaxSeedingMinutes();
+                }
 
                 if (seedingTimeLimit >= 0) {
-                    qDebug("Seeding Time: %d (limit: %d)", seedingTimeInMinutes, seedingTimeLimit);
```

Comment:
If you want to fix the warning just fix the typing on the format specifier for seedingTimeInMinutes to %lld (long long int)

Disregard if you find the print useless

Reply:
>Disregard if you find the print useless.

yeah I really want to remove it.